server/routerlicious/packages/lambdas/src/deli/lambdaFactory.ts

```diff
@@ -225,6 +225,22 @@ export class DeliLambdaFactory
 					closeType === LambdaCloseType.ActivityTimeout ||
 					closeType === LambdaCloseType.Error
 				) {
+					if (document?.isEphemeralContainer) {
+						// Call to historian to delete summaries
+						await gitManager.deleteSummary(false);
```

Comment:
should we `requestWithRetry/runWithRetry` on these 2 deletion operations?

Reply:
good idea..done